src/Uno.UI/UI/Xaml/Controls/TextBox/TextBox.Overrides.Android.cs

```diff
@@ -45,12 +45,19 @@ private class TextBoxStringBuilder : SpannableStringBuilder
 			private readonly ManagedWeakReference _owner;
 			private TextBox Owner => _owner.Target as TextBox;
 
+			private bool IsReadonly => Owner is not { } owner ? false : owner.IsReadOnly || !owner.IsTabStop;
```

Comment:
```suggestion
			private bool IsReadonly => Owner is { } owner && (owner.IsReadOnly || !owner.IsTabStop);
```

Consider also moving this property from `TextBoxStringBuilder` to `TextBox`, and use it in `UpdateTextBoxViewReadOnly` instead of duplicating the logic.

Reply:
Nice suggestion with the Pattern Matching!

About moving the logic: I did not want to add this logic on the `TextBox` since that would require changing the public API to add a new public property.

Reply:
@ajpinedam It doesn't have to be public I think. `private` might just work (or if it doesn't, then `internal` should)

#### `TextBox.Overrides.Android.cs`

On line 19, it would be `private bool IsNativeReadOnly => IsReadOnly || !owner.IstabStop;`

On line 56, `if (Owner is { IsNativeReadOnly: true }) return this;`

#### `TextBox.Android.cs`

On line 432, `var isReadOnly = IsNativeReadOnly;`

Reply:
I chose the name `IsNativeReadOnly` to not confuse it with Windows' IsReadOnly, since the new property checks both `IsReadOnly` and `IsTabStop`

Reply:
PR Updated. Added an internal property with the logic